src/components/organisms/settings/user/organizations/table.tsx

```diff
@@ -96,13 +98,11 @@ export const UserOrganizationsTable = () => {
 							<Td className="w-1/5 min-w-16">
 								<IconButton
 									className="mr-1"
-									disabled={user?.defaultOrganizationId === organization.id}
-									onClick={() =>
-										openModal(ModalName.deleteOrganization, {
-											name: organization.displayName,
-											id: organization.id,
-										})
+									disabled={
+										user?.defaultOrganizationId === organization.id ||
+										organization.currentMember?.role === MemberRole.admin
```

Comment:
If I understand correctly then correct like
```suggestion
										organization.currentMember?.role !== MemberRole.admin
```

Reply:
fixed